src/components/layout/NationalLayout.tsx

```diff
@@ -33,7 +33,7 @@ import siteText from '~/locale/index';
 import { WithChildren } from '~/types/index';
 
 import { INationalData } from '~/static-props/nl-data';
-import { useMenuState } from '~/utils/useMenuState';
+import { useMenuState } from '~/components/layout/useMenuState';
```

Comment:
Nipick; When you use feature local files, a relative import is cleaner and also clearly shows you that the dependency is local to the file.

```suggestion
import { useMenuState } from './useMenuState';
```